Recon/src/Recon/ParticleFlow.cxx

```diff
@@ -0,0 +1,405 @@
+#include "Recon/ParticleFlow.h"
+
+#include <vector>
+
+namespace recon {
+
+void ParticleFlow::configure(framework::config::Parameters& ps) {
+  // I/O
+  inputEcalCollName_ = ps.getParameter<std::string>("inputEcalCollName");
+  inputHcalCollName_ = ps.getParameter<std::string>("inputHcalCollName");
+  inputTrackCollName_ = ps.getParameter<std::string>("inputTrackCollName");
+  outputCollName_ = ps.getParameter<std::string>("outputCollName"); 
+  // Algorithm configuration
+  singleParticle_ = ps.getParameter<bool>("singleParticle"); 
+
+  // Calibration factors, from jason, temperary
+  std::vector<float> em1{250.0,750.0,1250.0,1750.0,2250.0,2750.0,3250.0,3750.0,4250.0,4750.0,5250.0,5750.};
+  std::vector<float> em2{1.175,1.02,0.99,0.985,0.975,0.975,0.96,0.94,0.87,0.8,0.73,0.665};
+  std::vector<float> h1{25.0,75.0,125.0,175.0,225.0,275.0,325.0,375.0,425.0};
+  std::vector<float> h2{8.44,7.38,7.76,8.535,9.47,10.45,10.47,9.71,8.87};
+  eCorr_ = new TGraph(em1.size(), em1.data(), em2.data());
+  hCorr_ = new TGraph(h1.size(), h1.data(), h2.data());
```

Comment:
in general when an object is `new`-ed, it should also have a `delete` somewhere in the end of the code to avoid memory leak. I didnt didnt find the delete, but maybe you do it somewhere else?

Reply:
indeed I missed this (thanks!) its now added to the process end